<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Superadminka</class>
 <widget class="QMainWindow" name="Superadminka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Театр</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="teatrButton0">
       <property name="text">
        <string>Мультфильмы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="teatrButton1">
       <property name="text">
        <string>Мокап</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Проекторы</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="prButton1">
       <property name="text">
        <string>Проекторы вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prButton0">
       <property name="text">
        <string>Проекторы выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Компьютеры</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pc1">
       <property name="text">
        <string>Компьютеры вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pc0">
       <property name="text">
        <string>Компьютеры выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Компьютеры</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="scriptButton1">
       <property name="text">
        <string>Скрипты вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scriptButton0">
       <property name="text">
        <string>Скрипты выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Реле</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="rele1">
       <property name="text">
        <string>Реле вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rele0">
       <property name="text">
        <string>Реле выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="park1">
       <property name="text">
        <string>Включить парк</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="park0">
       <property name="text">
        <string>Выключить парк</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Реле mqtt</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="mqtt_1">
       <property name="text">
        <string>mqtt вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mqtt_0">
       <property name="text">
        <string>mqtt выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Телевизоры sharp</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QPushButton" name="tv_sharp1">
       <property name="text">
        <string>TV вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tv_sharp0">
       <property name="text">
        <string>TV выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Телевизоры iiyama</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QPushButton" name="tv_iiyama1">
       <property name="text">
        <string>TV вкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tv_iiyama0">
       <property name="text">
        <string>TV выкл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
